src/controllers/refreshToken.controller.js

```diff
@@ -29,9 +29,9 @@ const refresh = async (req, res) => {
 		const accessToken = jwt.sign(
 			{ id: userToken.user.id, role: userToken.user.role },
 			process.env.ACCESS_TOKEN_SECRET,
-			{ expiresIn: '600s' }
+			{ expiresIn: '120s' }
```

Comment:
why did you change the expiration milliseconds?

Reply:
We were using that value to tackle the change of timeout while we were working on the integration since we had no refresh token access. I changed it because merging this PR will fix the issue and all requests can be intercepted automatically